sklearn/utils/__init__.py

```diff
@@ -32,8 +64,8 @@
            "check_consistent_length", "check_X_y", 'indexable',
            "check_symmetric", "indices_to_mask", "deprecated",
            "cpu_count", "Parallel", "Memory", "delayed", "parallel_backend",
-           "register_parallel_backend", "hash", "effective_n_jobs",
-           "resample", "shuffle"]
+           "register_parallel_backend", "joblib_hash", "effective_n_jobs",
```

Comment:
I agree `hash` is a bad variable name, but we can't rename it without a deprecation cycle now.

Reply:
I thought it was rename before this PR. My bad.